doc/fta-construction-algorithm.md

```diff
@@ -39,11 +39,21 @@ BEGIN
 END
 ```
 
-## Example 1
+## Example 1 - Or Behaviour
 Demonstrates transformation of OR behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree](https://user-images.githubusercontent.com/18463762/134146813-8a257931-26bb-4350-b954-c29ac7601c9c.png)
 
-## Example 2
+## Example 2 - And Behaviour
 Demonstrates transformation of AND behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree-example-2](https://user-images.githubusercontent.com/18463762/137472141-065769d1-8f6c-4267-ac5e-4fc85c16a0c4.png)
 
+## Example 3 - Live Demo
+### FTA EXamples
```

Comment:
```suggestion

```

Reply:
The section is talking about FTA, so I guess it does not make sense to repeat it again ... moreover don't see reason to have plural here